### PR TITLE
[Mission] CoP 2-3 Oil Lamp Location Fixes

### PR DESCRIPTION
**Pull request by Frankie-hz**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

## What does this pull request do?

Fixes Oil Lamp locations from South to North in Phomiuna Aqueducts.

## Steps to test these changes

1) Type: !pos -63 -26 43
2) Click Oil Lamps in order from South to North to get the corresponding messages. During a specific day a set of two lamps should open the next door.

**Order:**
Opens door to south room.
Fire
Earth
Water
Wind
Ice
Thunder (Lightning)
Light
Darkness
Opens door to north room.

**Solutions for door**
Firesday: Fire and Ice
Iceday: Ice and Wind
Windsday: Wind and Earth
Earthsday: Earth and Thunder
Lightningsday: Thunder and Water
Watersday: Water and Fire
Lightsday: Light and Dark
Darksday: Dark and Light

### CODE REVIEW

File: scripts/zones/Phomiuna_Aqueducts/npcs/_0rn.lua [hunk 1]
```diff
@@ -14,18 +14,14 @@ entity.onTrigger = function(player, npc)
 
     local DoorOffset = npc:getID()
```
Comment (The-Aerec) [inline scripts/zones/Phomiuna_Aqueducts/npcs/_0rn.lua]: doorOffset

Can't start local vars with capitals: see camel case

File: scripts/zones/Phomiuna_Aqueducts/npcs/_0rn.lua [hunk 2]
```diff
@@ -14,18 +14,14 @@ entity.onTrigger = function(player, npc)
...
 
-    player:messageSpecial(ID.text.LAMP_OFFSET+2) -- water lamp
-    npc:openDoor(7) -- lamp animation
+    player:messageSpecial(ID.text.LAMP_OFFSET+7) -- Dark lamp
+    npc:openDoor(7) -- Lamp animation
 
     local day = VanadielDayOfTheWeek()
 
-    if (day == xi.day.WATERSDAY) then
```
Comment (The-Aerec) [inline scripts/zones/Phomiuna_Aqueducts/npcs/_0rn.lua:22]: No parenthesis on single conditions like this

File: scripts/zones/Phomiuna_Aqueducts/npcs/_0rn.lua [hunk 3]
```diff
@@ -14,18 +14,14 @@ entity.onTrigger = function(player, npc)
...
-        if (GetNPCByID(DoorOffset+7):getAnimation() == 8) then -- lamp fire open?
```
Comment (The-Aerec) [inline scripts/zones/Phomiuna_Aqueducts/npcs/_0rn.lua:23]: same

File: scripts/zones/Phomiuna_Aqueducts/npcs/_0rn.lua [hunk 4]
```diff
@@ -14,18 +14,14 @@ entity.onTrigger = function(player, npc)
...
-            GetNPCByID(DoorOffset-2):openDoor(15) -- Open Door _0rk
-        end
-    elseif (day == xi.day.LIGHTNINGDAY) then
```
Comment (The-Aerec) [inline scripts/zones/Phomiuna_Aqueducts/npcs/_0rn.lua:26]: same

File: scripts/zones/Phomiuna_Aqueducts/npcs/_0rn.lua [hunk 5]
```diff
@@ -14,18 +14,14 @@ entity.onTrigger = function(player, npc)
...
-        if (GetNPCByID(DoorOffset+2):getAnimation() == 8) then -- lamp lightning open?
-            GetNPCByID(DoorOffset-2):openDoor(15) -- Open Door _0rk
+    if (day == xi.day.LIGHTSDAY or day == xi.day.DARKSDAY) then
```
Comment (The-Aerec) [inline scripts/zones/Phomiuna_Aqueducts/npcs/_0rn.lua]: same

File: scripts/zones/Phomiuna_Aqueducts/npcs/_0rn.lua [hunk 6]
```diff
@@ -14,18 +14,14 @@ entity.onTrigger = function(player, npc)
...
+        if (GetNPCByID(DoorOffset+1):getAnimation() == 8) then -- Is the Light lamp open?
```
Comment (The-Aerec) [inline scripts/zones/Phomiuna_Aqueducts/npcs/_0rn.lua]: same - repeat for the other files

File: scripts/zones/Phomiuna_Aqueducts/npcs/_0rn.lua (hunk 1 above)
Comment (The-Aerec) [reply scripts/zones/Phomiuna_Aqueducts/npcs/_0rn.lua]: same for all the other files

**Review by The-Aerec — COMMENTED** [review]
Hi: Styling / Ci comments

File: scripts/zones/Phomiuna_Aqueducts/npcs/_0rn.lua [hunk 7]
```diff
@@ -12,20 +12,16 @@ end
...
     local day = VanadielDayOfTheWeek()
 
-    if (day == xi.day.WATERSDAY) then
-        if (GetNPCByID(DoorOffset+7):getAnimation() == 8) then -- lamp fire open?
-            GetNPCByID(DoorOffset-2):openDoor(15) -- Open Door _0rk
-        end
-    elseif (day == xi.day.LIGHTNINGDAY) then
-        if (GetNPCByID(DoorOffset+2):getAnimation() == 8) then -- lamp lightning open?
-            GetNPCByID(DoorOffset-2):openDoor(15) -- Open Door _0rk
+    if day == xi.day.LIGHTSDAY or day == xi.day.DARKSDAY then
```
Comment (The-Aerec) [inline scripts/zones/Phomiuna_Aqueducts/npcs/_0rn.lua]: Multi conditional statements should look like:

`
if 
    day == xi.day.LIGHTSDAY or 
    day == xi.day.DARKSDAY
then
`

**Review by zach2good — APPROVED** [review]
This is probably fine if someone playtests it

**Comment by zach2good** [comment]
(and squash)

**Review by Xaver-DaRed — APPROVED** [review]
Tested and confirmed working as intended

**Pull request merged**